modules/apps/frontend-js/frontend-js-test/src/testFunctional/paths/TranslationManager.path

```diff
@@ -27,7 +27,7 @@
 </tr>
 <tr>
 	<td>MANAGE_TRANSLATIONS</td>
-	<td>//button[contains(@class,'dropdown-item')]//div[contains(@class,'autofit-section')and text()='Manage Translations']</td>
+	<td>//button[contains(@class,'dropdown-item')]//div[contains(@class,'autofit-section')and text()='Manage Translations'] | //div[contains(@class,'overlay-focused')]//button[contains(@id,'manage-translations')]</td>
```

Comment:
It's better not to add "|" in one path , as if it is changed in the future, it's not easy to maintain. Create a new one if needed.

And `//div[contains(@class,'overlay-focused')]//button[contains(@id,'manage-translations')]` this part seems duplicate as Translation#TRANSLATION_MANAGE_BUTTON, can you confirm if this works in this case?